<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>delete_movie</class>
 <widget class="QWidget" name="delete_movie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	color: #EEE;
	background-color: #111;
}

#delete_movie {
	background-color: qlineargradient(spread:pad, x1:0.637, y1:0.142364, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 86, 86, 207));
}

#title {
	background-color: transparent;
	font-size: 26px;
	color: rgba(13, 83, 170, 0.9);
}

#error {
	background-color: transparent;
	color: rgba(255, 0, 0, 0.8);
}

QLabel {
	font-size: 20px;
}

QLineEdit, QDateTimeEdit {
	border-radius: 10px;
	background-color: rgba(20, 20, 20, 0.75);
	padding: 5px 10px;
	border:  1px solid rgba(255, 255, 255, 0.25);
	border-bottom-color: rgba(110, 110, 255, 0.25);
	font-size: 16px;
}
QLineEdit:focus {
	border-bottom-color: #0d53aa;
}

QPushButton {
	border-radius: 10px;
	padding: 5px 10px;
	border: 1px solid #0d53aa ;
	background-color: #0d53aa; 
	font-weight: 900;
	font-size: 16px;
}
QPushButton:hover {
	background-color: red; 
	border-color: red;
}

#cancel_btn {
	border-color: rgba(84, 84, 84, 0.75);
	background-color: rgba(84, 84, 84, 0.75);
}</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>20</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cinemabooking Dashboard</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_btn">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>330</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Movie Name</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="showDate">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>230</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
     <year>2100</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd HH:mm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteUser_page">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>363</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 10px;</string>
   </property>
   <property name="text">
    <string>Delete User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_btn">
   <property name="geometry">
    <rect>
     <x>589</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editUser_page">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>435</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 10px;</string>
   </property>
   <property name="text">
    <string>Edit User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addMovie_page">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>147</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 10px;</string>
   </property>
   <property name="text">
    <string>Add movie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>451</x>
     <y>330</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editMovie_page">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>291</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 10px;</string>
   </property>
   <property name="text">
    <string>Edit Movie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteMovie_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>219</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 10px;</string>
   </property>
   <property name="text">
    <string>Delete movie</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>showDate</tabstop>
  <tabstop>delete_btn</tabstop>
  <tabstop>cancel_btn</tabstop>
  <tabstop>logout_btn</tabstop>
  <tabstop>addMovie_page</tabstop>
  <tabstop>deleteMovie_btn</tabstop>
  <tabstop>editMovie_page</tabstop>
  <tabstop>deleteUser_page</tabstop>
  <tabstop>editUser_page</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
